sdk/core/Microsoft.Extensions.Azure/src/Microsoft.Extensions.Azure.csproj

```diff
@@ -30,10 +28,15 @@
     <PackageReference Include="Microsoft.Extensions.Configuration.Binder" />
   </ItemGroup>
 
-  <ItemGroup>
+  <!-- Include Azure.Identity reference based on the way we reference Azure.Core -->
+  <ItemGroup Condition="'$(UseProjectReferenceToAzureCore)'=='true'">
```

Comment:
Do we currently have any other similar dependencies to Identity like this? If so should we generalize it like we did with Azure.Core?

Reply:
No, this one is the only case. If anything appears I'll generalize.